libnmstate/nm/bridge.py

```diff
@@ -90,7 +93,16 @@ def create_port_setting(options, base_con_profile):
         elif key == LB.Port.STP_PATH_COST:
             port_setting.props.path_cost = val
         elif key == LB.Port.VLAN_SUBTREE:
-            raise NmstateNotImplementedError
+            trunk_tags = val.get(LB.Port.Vlan.TRUNK_TAGS)
+            tag = val.get(LB.Port.Vlan.TAG)
+            enable_native_vlan = val.get(LB.Port.Vlan.ENABLE_NATIVE)
+            port_vlan_config = PortVlanFilter(
+                trunk_tags, tag, enable_native_vlan
+            ).to_nm()
```

Comment:
nit: Can you please first create the the object and only then call its method? I find it hard to follow this way.